content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,255 @@
+
+This experiment is then associated with the values found in parameters file
+(`params.yaml`), and other dependencies (`data/images/`) with these produced
+metrics.
+
+<details>
+
+### 📖 If you used `dvc repro` before
+
+Earlier versions of DVC uses `dvc repro` to run the pipeline. If you already
+have a DVC project, you may already be using `dvc repro`.
+
+In DVC 2.0 `dvc exp run` supersedes `dvc repro`. We use `dvc repro` to run the
+pipeline as found in the <abbr>workspace</abbr>. All the parameters and
+dependencies are retrieved from the current workspace. It doesn't use any
+special objects to track the experiments or associate parameters with metrics.
+When you have large number of experiments that you don't want to commit all to
+Git, it's better to use `dvc exp run`. It allows to change the parameters
+quickly, can track the history of artifacts and has facilities to compare these
+experiments easily.
+
+`dvc repro` is still available to run the pipeline when these extra features are
+not needed.
+
+</details>
+
+## ⏲ Running the experiment by setting parameters
+
+Now let's do some more experimentation.
+
+DVC allows to update the parameters defined in the pipeline without modifying
+the files manually. We use this feature to set the convolutional units in
+`train.py`.
+
+```dvc
+$ dvc exp run --set-param model.conv_units=24
+...
+Reproduced experiment(s): exp-7b56f
+Experiment results have been applied to your workspace.
+...
+```
+
+<details>
+
+## 👟👟 Run multiple experiments in parallel
+
+Instead of running the experiments one-by-one, we can define them to run in a
+batch. This is especially handy when you have long running experiments.
+
+We add experiments to the queue using the `--queue` option of `dvc exp run`. We
+also use `-S` (`--set-param`) to set a value for the parameter.
+
+```dvc
+$ dvc exp run --queue -S model.conv_units=32
+Queued experiment '3cac8c6' for future execution.
+$ dvc exp run --queue -S model.conv_units=64
+Queued experiment '23660b6' for future execution.
+$ dvc exp run --queue -S model.conv_units=128
+Queued experiment '6591a57' for future execution.
+$ dvc exp run --queue -S model.conv_units=256
+Queued experiment '9109ea9' for future execution.
+```
+
+Next, run all (`--run-all`) queued experiments in parallel. You can specify the
+number of parallel processes using `--jobs`:
+
+```dvc
+$ dvc exp run --run-all --jobs 2
+```
+
+</details>
+
+## ↔️ Comparing experiments
+
+The experiments are run several times with different parameters. We use
+`dvc exp show` to compare all of these experiments. This command presents the
+parameters and metrics produced in experiments in a nicely formatted table.
+
+```dvc
+$ dvc exp show
+```
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
+┃ white:**Experiment**    ┃ white:**Created**┃    yellow:**loss** ┃    yellow:**acc** ┃ blue:**train.epochs** ┃ blue:**model.conv_units** ┃
```

Comment:
Looks like certain parts of the table are missing some white space, causing the table to not align correctly. 

```suggestion
┃ white:**Experiment**              ┃ white:**Created**      ┃ yellow:**loss**    ┃ yellow:**acc**    ┃ blue:**train.epochs** ┃ blue:**model.conv_units** ┃
```